docs/core/distribution-packaging.md

```diff
@@ -24,8 +24,14 @@ When installed, .NET consists of several components that are laid out as follows
 │   └── fxr                                       (*)
 │       └── <fxr version>        (2)
 ├── sdk                                           (*)
-│   ├── <sdk version>            (3)
-│   └── NuGetFallbackFolder      (4)              (*)
+│   └── <sdk version>            (3)
+├── sdk-manifests                (4)              (*)
+│   └── <sdk version>
```

Comment:
This folder isn't exactly the SDK version.  It is the feature band version.  So for an SDK version such as 6.0.102, this folder would still be versioned at 6.0.100.